images/melange/image.yaml

```diff
@@ -1,4 +1,8 @@
 versions:
+  - apko:
+      config: configs/0.apko.yaml
```

Comment:
Will this also tag them with :0?

Reply:
yes. and actually, even if we rename this, the way tagging works with extractTagsFrom and the version stemming, 0 would be added as a version tag anyway, since it is the major version technically

Reply:
we would have to add an option to ignore major / single digit tags inside apko